solr/core/src/java/org/apache/solr/cloud/CreateAliasCmd.java

```diff
@@ -68,34 +249,100 @@ public void call(ClusterState state, ZkNodeProps message, NamedList results)
     Thread.sleep(100);
   }
 
+  private Map<String, String> buildAliasMap(String routedField, String routingType, String tz, String increment, String maxFutureMs, ZkNodeProps collectionProps) {
+    Map<String, Object> properties = collectionProps.getProperties();
+    Map<String,String> cleanMap = properties.entrySet().stream()
+        .filter(stringObjectEntry ->
+            !"fromApi".equals(stringObjectEntry.getKey())
```

Comment:
These 3 constants here are unfortunate.... maybe they could be in a constant Set and checked here?

Reply:
yeah, this is stuff that the collection create code adds. I'll look if I can use constants from there and make a set out of them.

Reply:
Made my own constants & made set existing constants were too scattered/disorganized/nonexistant ... fixed in update